sounds/views.py

```diff
@@ -115,7 +115,10 @@ def sounds(request):
     popular_packs = Pack.objects.select_related('user').filter(created__gte=last_week).exclude(is_deleted=True).order_by("-num_downloads")[0:5]
     random_sound_id = get_sound_of_the_day_id()
     if random_sound_id:
-        random_sound = Sound.objects.bulk_query_id([random_sound_id])[0]
+        try:
+            random_sound = Sound.objects.bulk_query_id([random_sound_id])[0]
+        except IndexError:
```

Comment:
I think we probably don't want to do a lot of things here that would complicate code. The nice thing about random sound of the day is that it gets fixed automatically the next day anyway. The issue today is that Freesound was broken because of that... 

For now what I would suggest is to make sure that running the command to generate new random sounds will fix the issue (by filling in the gap and clearing the cache) so we have an easy way to manually fix that.